specification/scvmm/resource-manager/readme.md

```diff
@@ -39,6 +39,15 @@ input-file:
   - Microsoft.ScVmm/preview/2020-06-05-preview/scvmm.json
 ```
 
+### Tag: package-2022-05-21-preview
+
+These settings apply only when `--tag=package-2022-05-21-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2022-05-21-preview'
+input-file:
+  - Microsoft.ScVmm/preview/2022-05-21-preview/scvmm.json
+```
+
```

Comment:
Please remove the following line as well.

Line 67:    - repo: azure-sdk-for-ruby

Reply:
And line 83:

## Ruby
See configuration in [readme.ruby.md](./readme.ruby.md)

Reply:
Updated.